test/literate_tabbed.litcoffee

```diff
@@ -69,13 +70,15 @@ if true
 
 	test "should ignore code blocks inside HTML", ->
 		eq executed, false
+		testsCount++
 
 ---
 
-*   A list item with a code block:
+*   A list item followed by a code block:
 
-		test "basic literate CoffeeScript parsing", ->
-			ok yes
+	test "basic literate CoffeeScript parsing", ->
```

Comment:
I see your point, but if someone is executing a .litcoffee file, well, they’re executing code. They shouldn’t be surprised if code blocks execute, even if they mistakenly presume a particular code block is a commentary code block. People who have code-execution rights need to have some responsibility for what they’re doing.